### ISSUE TITLE
Literal suffix `lu` and `llu` not accepted

### ISSUE THREAD

**Issue by andrewvaughanj** [ltcmelo/psychec#30]
#### GCC version:
```bash
gcc (SUSE Linux) 10.2.1 20201202 [revision e563687cf9d3d1278f45aaebd03e0f66531076c9]
```

#### clang version:
```bash
clang version 11.0.1
```

#### Output of `gcc -Werror -Wall -Wextra -Wpedantic -pedantic -std=c11 -c $FNAME`
```bash
(return code 0)
```

#### Output of `clang -Werror -Wall -Wextra -Wpedantic -pedantic -std=c11 -c $FNAME`
```bash
(return code 0)
```

#### Output of `cnip $FNAME`
```bash
suffix.c:0:23 error: expected expression got `#error#'
int foo(void) { return 0xcccclu; }
                       ^

(return code 0)
```

#### Content of file
```bash
int foo(void) { return 0xcccclu; }
```

#### `psychec` version
```bash
commit d10dc1ddfb7ea845f59274a4d6ca5d021ab31423
Author: Leandro T. C. Melo <LTCMELO@GMAIL.COM>
Date:   Tue Mar 9 21:18:30 2021 -0300

    fix postfixing on compound literal
```

**Comment by andrewvaughanj** [comment]
Same for `llu`:

#### GCC version:
```bash
gcc (SUSE Linux) 10.2.1 20201202 [revision e563687cf9d3d1278f45aaebd03e0f66531076c9]
```

#### clang version:
```bash
clang version 11.0.1
```

#### Output of `gcc -Werror -Wall -Wextra -Wpedantic -pedantic -std=c11 -c $FNAME`
```bash
(return code 0)
```

#### Output of `clang -Werror -Wall -Wextra -Wpedantic -pedantic -std=c11 -c $FNAME`
```bash
(return code 0)
```

#### Output of `cnip $FNAME`
```bash
suffix.c:0:23 error: expected expression got `#error#'
int foo(void) { return 0xccccllu; }
                       ^

(return code 0)
```

#### Content of file
```bash
int foo(void) { return 0xccccllu; }
```

#### `psychec` version
```bash
commit d10dc1ddfb7ea845f59274a4d6ca5d021ab31423
Author: Leandro T. C. Melo <LTCMELO@GMAIL.COM>
Date:   Tue Mar 9 21:18:30 2021 -0300

    fix postfixing on compound literal
```

**Comment by ltcmelo** [comment]
Thanks @andrewvaughanj , fixed with 490a43bc799d95cf2049fd4461b32af74747d3dd.
